ts/session/constants.ts

```diff
@@ -99,18 +99,12 @@ export const REACT_LIMIT = 6;
 
 export const UPDATER_INTERVAL_MS = 10 * DURATION.MINUTES;
 
-/**
- * Start create groups as new at this time (currently Thursday March 20th 09:00 AEDT)
- */
-const START_CREATE_NEW_GROUP = 1742421600000;
-
 /**
  * Mark legacy groups readonly at this time (currently Thursday April 3rd 09:00 AEDT)
  */
 const LEGACY_GROUP_READONLY = 1743631200000;
```

Comment:
I guess later we just make this behaviour the default without relying on the timestamp?

Reply:
change made